### ISSUE TITLE
VNC connection failed. Connection refused

### ISSUE THREAD

**Issue by valadina223** [CypherpunkArmory/UserLAnd#1073]
# Describe the bug

Installed UserLAnd, installed Ubuntu, installed LXDE. After finishing LXDE the terminal 'told' me to restart the service because installation was done. I did so but now cannot connect to LXDE. Starting it via Sessions -> VNC apps gives me an error message.

First error message is: 
`VNC connection failed! null`

If I try to connect again it says: 
`VNC connection failed! failed to connect to /127.0.0.1 (port 5951) from /::(port37216): connection failed: ECONNREFUSED (Connection refused)`

I can however connect to the basic Ubuntu setup, there I get the screen and the terminal.

# Steps to reproduce the behavior:

1) Install UserLAnd
2) Set up Ubuntu within UserLAnd
3) Set up LXDE within UserLAnd
4) Try to connect to LXDE

# Device Information

Device: Wileyfox  
Android LineageOS 16
UserLAnd Version 2.6.3 
Device is not rooted

# Log says:

```[H[J
Can't find file /home/ubuntu/.vnc/localhost:51.pid
You'll have to kill the Xtightvnc process manually


New 'X' desktop is localhost:51

Starting applications specified in /home/ubuntu/.vnc/xstartup
Log file is /home/ubuntu/.vnc/localhost:51.log

xterm: fatal IO error 11 (Resource temporarily unavailable) or KillClient on X server "localhost:51"
```

**Comment by corbinlc** [comment]
Does #1074 mean that `xfce4` is now working for you?

**Comment by valadina223** [comment]
Yes, xfce4 was working right from the beginning. But, there have been those problems with certain apps mentioned in #1074.

**Comment by corbinlc** [comment]
Oh sorry, this is for LXDE.  I was thinking the two reports were contradictory...they are not.

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
